components/filter/filter-dimension-set.js

```diff
@@ -1,9 +1,16 @@
 import { html, LitElement } from 'lit';
 
+export const EmptyStateType = {
```

Comment:
Minor: Since this isn't used in here anymore, we could maybe move all the logic to one file or the other. For example, we could clean things up a bit by just sending the type as part of the `_getEmptyState` response.  Something like:
```
getSearchEmptyState() {
	return this._getEmptyState(this._searchEmptyStateSlot, 'search');
}

getSetEmptyState() {
	return this._getEmptyState(this._setEmptyStateSlot, 'set');
}

_getEmptyState(emptyStateSlot, emptyStateType) {
...
		return {
			actionHref: emptyState.actionHref,
			actionText: emptyState.actionText,
			description: emptyState.description,
			type: emptyStateType
		};
}
```
And then in filter have:
```
_createEmptyState(emptyState, dimensionKey) {
  <d2l-empty-state-action-button
	  @d2l-empty-state-action="${this._handleEmptyStateAction}"
	  data-dimension-key="${dimensionKey}"
	  data-type="${emptyState.type}"
```

Or even keep the const and pass `EmptyStateType.Search` and `EmptyStateType.Set` into `_getEmptyState`, but not have to export it anymore. Not a big deal either way, but it doesn't seem like `d2l-filter` really _cares_ what the string is, just that it needs to pass something to the consumer for differentiating.

Reply:
I missed that, that sounds reasonable. I like the idea of moving the type into the `_getEmptyState()` response